CHANGELOG.md

```diff
@@ -11,6 +11,19 @@ The present file will list all changes made to the project; according to the
 - `Link ITIL Object` and `Unlink ITIL Object` massive actions for Tickets, Changes, and Problems.
 - Group approval requests (any user from group can approve).
 - Satisfaction surveys for Changes
+- New right for assigning service levels to ITIL Objects (UPDATE right also still allows this).
+- New right for impersonation of users with less rights (Config UPDATE right also still allows this globally).
+- Marketplace availability configuration.
+- Toast popup message location configuration.
+- Datacenter room grid size configuration (per room).
+- Approval reminder automatic action.
+- Reservation massive actions.
+- `alias` and `code` fields for Locations.
+- Profile cloning.
+- Associated elements for recurring ITIL Objects.
+- Processes and Environment Variable tabs for automatically inventoried assets.
+- Log viewer for logs in `files/_log` directory.
+- Custom palette/theme support (uses `files/_themes` directory by default).
```

Comment:
this one is not merged yet :p (but we can keep)

Reply:
#11669 was merged.
Was there another related PR or was the original not meant to be merged yet?